controllers/comments_controller.js

```diff
@@ -16,11 +20,23 @@ exports.getCommentsByArticleId = (req, res, next) => {
 exports.postComment = (req, res, next) => {
   const { article_id } = req.params;
   insertComments(req.body, article_id)
-   .then((comment) => {
-   res.status(201).send({ comment });
+    .then((comment) => {
+      res.status(201).send({ comment });
     })
     .catch((err) => {
       next(err);
     });
 };
-
+exports.deleteComment = (req, res, next) => {
+  const { comment_id } = req.params;
+  removeCommentById(comment_id)
+    .then(() => {
+      res.status(204).send();
+    })
+    .catch((err) => {
+      if (err.message === "Not found") {
```

Comment:
You should be allowing the error to be passed to the catch and dealt with in the error handlers in app.js instead of sending the message here. Also, why are you using .json?